Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/UpgradeTool/Scanner.cpp

```diff
@@ -52,6 +54,8 @@ namespace ScannerCpp
 
 namespace ScriptCanvasEditor
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether

Reply:
This `using namespace ScriptCanvas;` is still at namespace scope